### DISCUSSION TITLE
Bug with Operational Risk

### DISCUSSION THREAD

**Discussion by danje57** [monarc-project/MonarcAppFO#415] [Q&A] [answered]
Dear all,

Yesterday I created some Operational Risks.

However, I made some mistakes in the Description field. I saw it too late :-/ and I've created more than 100 risks.

I've modified the Description fields using the Knowledge base / Operational Risk editor option.

So now, in the Knowledge base everything is OK.

However, In the risk analysis when I scroll to the risk, the popup description is still the old one.

I tried to logoff, clean up my browser cache however in the risk analysis description is always the wrong one.

Is it possible to force the description? Or I should recreate all risks....

Thanks in advance for your help.

Regards

**Comment by ruslanbaidan** [comment]
Hello,

Could you tell us if you meant the "Risk description" in the operational risks list (sheet) of the analysis, where it not updated?
The case is that the "Risk description" in the operational risks list of an analysis is the value of "Label" field filled in the Knowledge Base.

In any case the Label modification in the Knowledge Base sometimes leads to operational risk duplication.
We are going to fix the problem.

Thank you.

Regards.

**Comment by danje57** [comment]
Hi,

In Operational Risk, the Label field doesn't exist:

![image](https://user-images.githubusercontent.com/16591723/155114508-ad4115bc-30fa-459e-8f3a-68b91850e342.png)

In the Knowledge base, We can fill the description field, which is the information that appear in the information bubble:

![image](https://user-images.githubusercontent.com/16591723/155115147-097f1999-b997-40fd-a1b5-e59c2108ff0e.png)

As you can see, the Message in the information bubble is the old one when I made the mistake.


Regards,

And thanks again!

**Reply by ruslanbaidan** [reply]
Hi,

In the Operational Risk table (your first screenshot), the Label field is taken from the "Label" field (second screenshot) of the Knowledge base.
In other words: Risk description === Label.
Could you try to modify Label instead of Description in the Knowledge base.

**Comment by danje57** [comment, marked as answer]
Unfortunately I'm sure about my explanation:

When I made the mistake, i made this:

Code ==> Same name as Risk Description but with underscore to replace the space

Risk description: ID-Risk name with space instead of underscore.

and Description, the same as Risk description but without the ID.

So I can assert that the field Description is the one used as Information Bubble to describe the operational risk :)


Si I assume that my mistake is irremediable....


Regars and thanks :)

**Reply by ruslanbaidan** [reply]
I'm marked on your screenshots the fields to be modified if you need to change the risks description of your analysis.
To update the "Risk description" filed in the list, you need to change the "Label" in the "Knowledge Base" (image is attached).

Could you tell if it is what you wanted to achieve?

![RiskDescrChange](https://user-images.githubusercontent.com/3246171/159703688-543f3849-642b-4a96-89e7-1ebfcc4d3801.png)

Thanks you.

**Reply by danje57** [reply]
I solved the issue by deleting all operational risks and recreating from scratch using import function.

As when I changed it directly on the knowledge base, the change was not reflected in the analysis.

However for me it's not a issue now ;)

**Reply by ruslanbaidan** [reply]
Ok.
I've just made a record to show that the functionality working.

https://user-images.githubusercontent.com/3246171/159873757-f5c2dfea-3868-48d0-9da8-72483f467b82.mov

**Reply by danje57** [reply]
Thanks for that!

You should keep this video in the gallery!

Thanks for your help.
